### PR TITLE
Blog routing

### PR DESCRIPTION
**Pull request by alodahl**
I rearranged the blog components to create a page for each blog article, and routed each with its own url.  Unfortunately, the single-page model does not work in gatsby, so returning to the landing page via #team in the navbar, for example, doesnt work. 

 If we approve this, I can remove "blog" from the navbar before the merge, and I'll work on it more once we transfer to unstack.

In the same vein, browserRouter isnt working in production the way I am using it in gatsby, so you cant see it!  Ack!  So here are pictures of what I made:
screen shot 2018-09-11 at 11 16 30 am
screen shot 2018-09-11 at 11 16 51 am

### CODE REVIEW

File: gatsby-site/src/components/blog-article/BlogArticle.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { articles } from '../blog-list/BlogList.js';
+import styles from './BlogArticle.css'
+
+export default function BlogArticle(props) {
+  const path = window.location.pathname;
```
Comment (heyellieday) [inline gatsby-site/src/components/blog-article/BlogArticle.js]: This isn't going work when we try to server side render things. Could we use a prop to get this info?

File: gatsby-site/src/pages/blog/index.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import BlogArticle from '../../components/blog-article/BlogArticle'
+import BlogMenu from '../../components/blog-menu/BlogMenu'
+import BasicPage from '../../components/BasicPage'
+import {
+  BrowserRouter,
+  Route
+} from 'react-router-dom'
+
+export default class BlogMenuPage extends React.Component {
```
Comment (heyellieday) [inline gatsby-site/src/pages/blog/index.js]: Is BlogMenu the right word for this component? Maybe it should be BlogList or something?

File: gatsby-site/src/components/blog-article/BlogArticle.js (hunk 1 above)
Comment (alodahl) [reply gatsby-site/src/components/blog-article/BlogArticle.js]: Sure thing! I’ll also take out the routing until I can route it in unstack.

File: gatsby-site/src/pages/blog/index.js (hunk 2 above)
Comment (alodahl) [reply gatsby-site/src/pages/blog/index.js]: I already have a blogList, but I’ll think of something. Maybe the blogList, which is just data, can be blogData, then the menu page can be blogList?

**Pull request closed without merging**